chromium_src/chrome/browser/ui/views/permission_bubble/permission_prompt_bubble_view.cc

```diff
@@ -87,10 +95,88 @@ void AddAdditionalWidevineViewControlsIfNeeded(
     views::BubbleDialogDelegateView* dialog_delegate_view,
     const std::vector<permissions::PermissionRequest*>& requests) {}
 #endif
+
+// Custom combobox, shows permission lifetime options and applies selected value
+// to all permissions currently visible in the bubble.
+class PermissionLifetimeCombobox : public views::Combobox,
```

Comment:
can you also mention in the comments that this currently a development stub?

Reply:
sure